linting/lint-csv.py

```diff
@@ -0,0 +1,35 @@
+import os
+import subprocess
+import sys
+
+
+CSV_PATH = os.path.join(os.path.dirname(__file__), '..')
```

Comment:
How about Pathlib?

Reply:
While the CI runs with 3.x, I don't think we should restrict local linting to 3.x.